rubocop-airbnb/lib/rubocop/cop/airbnb/continuation_slash.rb

```diff
@@ -14,7 +14,7 @@ def enforce_violation(node)
         alias on_send enforce_violation
         alias on_if enforce_violation
 
-        Util::ASGN_NODES.each do |type|
+        AST::Node::ASSIGNMENTS.each do |type|
```

Comment:
Is there a way we could make this change such that it uses whichever is available, so that it could work in both 2.1 and 2.6?

Reply:
Good idea! I'll push a solution soon